wren-modeling-py/Cargo.toml

```
@@ -5,12 +5,12 @@ edition = "2021"
 include = ["/src", "pyproject.toml", "Cargo.toml", "Cargo.lock"]
 
 [lib]
-name = "modeling_core"
+name = "wren_core"
```

```suggestion
name = "wren_core_py"
```
I think it's a Python adaptor. Maybe have a `py` suffix si better.

---

This name will be used in the Python codebase. When importing, it is weird if it is with the suffix py in the Python codebase.

---

I found datafusion-python also name it with the suffix but import without suffix.
https://github.com/apache/datafusion-python/blob/45a684445e25032961a7bb44ced3ce06f5ed9e6d/Cargo.toml#L64
Do you have any idea for it?

---

hmmm, Let me try again.

---

Cool, it seems only to follow `[tool.maturin.module-name]` in `pyproject.toml`. I will change `[lib.name]` to `wren_modeling_py` to be the same as the package name. And keep the module name `wren_core`.